src/main/resources/application.yml

```
@@ -0,0 +1,77 @@
+      default-client:
+        client-id: default-client
+        client-secret: secret
+        redirect-uris:
+          - http://127.0.0.1:8080/login/oauth2/code/default-client
+          - http://localhost:5173/callback
+        scopes:
+          - openid
+          - profile
+          - message.read
+          - message.write
+        require-consent: true 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)